onnxruntime/core/optimizer/embed_layer_norm_fusion.cc

```diff
@@ -699,6 +699,16 @@ Status EmbedLayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_l
                                                 {layer_norm_node.MutableOutputDefs()[0], reduce_sum_node.MutableOutputDefs()[0]},
                                                 {}, kMSDomain);
 
+    // Get attribute "epsilon" from "LayerNormalization" node if available. Else, default value
+    // will be used.
+    NodeAttributes ln_attrs = layer_norm_node.GetAttributes();
+    NodeAttributes::const_iterator epsilon = ln_attrs.find("epsilon");
+    if (epsilon != ln_attrs.end()) {
+      embed_layer_norm_node.AddAttribute("epsilon", epsilon->second);
+    } else {
+      embed_layer_norm_node.AddAttribute("epsilon", DEFAULT_EMBED_EPSILON);
```

Comment:
I don't think you need it. epsilon already has a default value in the schema definition

Reply:
As the discussion with @tianleiwu earlier, always adding epsilon attribute in graph during fusion gives us more flexibility because it will continue to support existing models even if we change default epsilons in the future.